docs/HowToGuides/FAQ.md

```diff
@@ -133,5 +133,45 @@ git rebase --continue
 
 ### How do I clean up my git history?
 
-TODO: Link to a beginner-friendly external resource, or (less preferably)
-describe basic usage of rebase here.
+Git's history can sometimes become cluttered with many small commits. 
+Fortunately, Git has a feature called `rebase` that allows you to clean up your commit history. 
+If you want to learn more, 
+[GitHub - About Git rebase](https://docs.github.com/en/get-started/using-git/about-git-rebase) 
+provides a comprehensive overview of `rebase`.
+Here's a small gist that goes through the basics on how to use it:
+
+1. Begin an interactive rebase: Use `git rebase -i HEAD~N`, where `N` is the number of commits 
```

Comment:
It might be worth to mention that this should only be performed on the commits that are not yet pushed, rebasing existing commits would block merging because we don't allow force pushes to the repository.

Reply:
Just one review ago. 
I had changed the sentence  
> It's important to note that you should only rebase commits that have not been pushed to a public branch.

Should I add a further reason? Or is this separate from these?

Reply:
Yeah, I think you move that to the top as a good disclaimer.

Reply:
I Fixed it. How about this🤔

Reply:
Looks great, thank you!